docs/src/mdx-components.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import {
+  Heading,
+  Text,
+  TextLink,
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+} from "@kiwicom/orbit-components";
+import { InlineCode, Pre } from "./components/Code";
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export const p = Text;
+export const h1 = (props: Props) => <Heading as="h1" type="display" {...props} />;
+export const h2 = (props: Props) => <Heading as="h2" type="title1" {...props} />;
+export const h3 = (props: Props) => <Heading as="h3" type="title2" {...props} />;
+export const h4 = (props: Props) => <Heading as="h4" type="title3" {...props} />;
+export const h5 = (props: Props) => <Heading as="h5" type="title4" {...props} />;
+export const h6 = (props: Props) => <Heading as="h6" type="title5" {...props} />;
+export const table = Table;
+export const thead = TableHead;
+export const tbody = TableBody;
+export const tr = TableRow;
+export const td = TableCell;
+export const th = TableCell;
+export const pre = Pre;
+export const inlineCode = InlineCode;
+export const a = TextLink;
```

Comment:
It'd be good to handle internal links separately from external links. It'd be enough to handle it with an `onClick` event with `navigate` from Gatsby for internal links.

Reply:
Good idea, I can also use Gatsby's `Link` component considering that `TextLink` supports `asComponent` 💪 I should also remember to pass `external` to `TextLink` for external links!